package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "puppeteer-screenshots",
+  "type": "module",
+  "scripts": {
+    "screenshots": "node screenshots/screenshots.js"
+  },
+  "dependencies": {
+    "puppeteer": "^24.4.0",
+    "puppeteer-core": "^24.4.0",
+    "xml2js": "^0.6.2"
+  }
+}
```

Comment:
Let's remove the screenshot tooling from this PR to make it smaller.

Reply:
This is still there.